README.md

```diff
@@ -348,23 +348,34 @@ Multiple commands are possible - they will be run in the order specified:
 $ spago build --before clear --before "notify-send 'Building'"
 ```
 
-If you want to run the program (akin to `pulp run`), just use `run`:
+If you want to run the program (akin to `pulp run`), just use `run`. The commands below are verbose until these values are defined in the configuration file:
```

Comment:
I don't think there's a need to mention the verbosity since we'll explain that in a second
```suggestion
If you want to run the program, just use `run`:
```